### PR TITLE
Fixes #29609 - fix initializer for rails 6

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by tbrisker** [comment]
failures are due to jquery-ui update that was just merged, can you please rebase?

**Comment by jlsherrill** [comment]
[test katello]

**Comment by mmoll** [comment]
angular-ui test failures unrelated (Github was broken) :green_heart:

**Comment by tbrisker** [comment]
[test katello] because even admins can't merge on red in katello

**Comment by tbrisker** [comment]
I'm afraid this could be an actual failure:
```
[2020-04-23T15:43:43.270Z] ActiveRecord::RecordNotFound: Couldn't find Setting with [WHERE "settings"."name" = $1]

**Comment by jlsherrill** [comment]
updated to make sure the settings are loaded prior to seeding.  I wasn't able to get db:seed to work properly against the test db manually at all, with or without this change (failed related to adding a permission to a default role), but i did test this with rails 6 and it worked, and seeding in the dev environment seemed to be doing things in the right order.   Lets see what jenkins does

**Comment by jlsherrill** [comment]
this is green... which i'm a little perplexed by.   let me try again

[test katello]

**Comment by jlsherrill** [comment]
[test katello]     timeout error

**Comment by jlsherrill** [comment]
[test katello] timeout error  again....

**Comment by jlsherrill** [comment]
[test katello]  unrelated test failures already fixed by @ianballou in master

**Comment by jlsherrill** [comment]
passed a 2nd time, lets try it one more to be 'safe'

[test katello[

**Comment by jlsherrill** [comment]
but this time i'll type it correctly

[test katello]

**Comment by jlsherrill** [comment]
[test katello]

 transient error... i think i'll open a pr for

**Comment by jlsherrill** [comment]
going ahead and merging
